data/mods/gen9ssb/abilities.ts

```diff
@@ -1194,6 +1194,45 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		flags: {},
 	},
 
+	// Two of Roses
+	aswesee: {
+		shortDesc: "Once per turn, when any active Pokemon has a stat boosted, this Pokemon has a 50%" +
+		" chance of copying it and a 15% chance to raise another random stat.",
+		name: "As We See",
+		onFoeAfterBoost(boost, target, source, effect) { // Opportunist
+			if (this.randomChance(1, 2)) {
+				if (effect?.name === 'As We See' || effect?.name === 'Mirror Herb' || effect?.name === 'Opportunist') return;
+				const pokemon = this.effectState.target;
+				const positiveBoosts: Partial<BoostsTable> = {};
+				let i: BoostID;
+				for (i in boost) {
+					if (boost[i]! > 0) {
+						positiveBoosts[i] = boost[i];
+					}
+				}
+				if (Object.keys(positiveBoosts).length < 1) return;
+				this.boost(positiveBoosts, pokemon);
+			}
+		},
+		onResidual(target, source, effect) {
+			if (this.randomChance(15, 100)) {
+				const stats: BoostID[] = [];
+				const boost: SparseBoostsTable = {};
+				let statPlus: BoostID;
+				for (statPlus in target.boosts) {
+					if (statPlus === 'accuracy' || statPlus === 'evasion') continue;
+					if (target.boosts[statPlus] < 6) {
+						stats.push(statPlus);
+					}
+				}
+				const randomStat: BoostID | undefined = stats.length ? this.sample(stats) : undefined;
+				if (randomStat) boost[randomStat] = 1;
+				this.boost(boost, target, target);
+			}
+		},
```

Comment:
This isn't moody

Reply:
It functionally is though, boosting a random stat every turn fits the bill so i modified it to fit this spec